include/aws/http/private/h2_frames.h

```diff
@@ -220,6 +221,7 @@ int aws_h2_encode_frame(
  * Attempt to encode a DATA frame into the output buffer.
  * AWS_OP_ERR is returned if encoder encounters an unrecoverable error.
  * body_complete will be set true if encoder reaches the end of the body_stream.
+ * body_stalled will be set true if body_stream didn't provide as much data as we would have liked.
```

Comment:
(Personal opinion) When I read this, I am actually a little bit confused about "we would have like". After reading the implementation, I know what it means. It might be helpful to clarify this a little bit more.

Reply:
```
* body_stalled will be true if aws_input_stream_read() stopped early (didn't
 * complete, though more space was available).
 ```